Src/coffeescript/cql-execution/src/elm/list.coffee

```diff
@@ -60,59 +51,46 @@ module.exports.SingletonFrom = class SingletonFrom extends Expression
     else if arg.length is 1 then return arg[0]
     else return null
 
-module.exports.IndexOf = class IndexOf extends UnimplementedExpression
-
-# TODO: Deconflict w/ definition in string.coffee
-# module.exports.Indexer = class Indexer extends UnimplementedExpression
-
-# TODO: Deconflict w/ definition in interval.coffee
-module.exports.In = class In extends Expression
+module.exports.IndexOf = class IndexOf extends Expression
   constructor: (json) ->
     super
+    @source = build json.source
+    @element = build json.element
 
   exec: (ctx) ->
-    [item, container] = @execArgs(ctx)
-
-    switch
-      when typeIsArray container
-        return item in container
-      when container instanceof ValueSet
-        return container.hasCode item
-
-# TODO: Deconflict w/ definition in interval.coffee
-module.exports.Contains = class Contains extends UnimplementedExpression
+    src = @source.exec ctx
+    el = @element.exec ctx
+    if not src? or not el? then return null
```

Comment:
Actually, that's a great point.  I was following the spec's guidance on `IndexOf` though:

> If either argument is null, the result is null.

So... we should leave it for now since it is according to spec, but I will add a ballot comment to consider changing it to allow for searching on null.  That said, I'm not sure there are too many use cases for it.